dogfooding/README.md

```diff
@@ -83,23 +91,54 @@ flutter doctor -v
 
 In the output go to the `Android toolchain` section and ensure that the JDK version is correct.
 
-6. Run the app
-Once the setup is complete, you can run the app using Flutter:
+### 5. Run the app
+
+Once the setup is complete, launch the app:
 
 ```bash
 ## Ensure you're in the dogfooding directory
 flutter run
 ```
 
-The app should now launch on your connected device or simulator.
+The app should now start on your connected device or simulator.
+
+### 6. Important notes
+
+#### Email-Based Login & User ID Conversion
+
+When logging in with an email, the user_id automatically replaces @ and . with underscores (_).
+For example:
+
+- Email: test@test.com
+- User ID: test_test_com
+
+This is crucial for testing direct calls, as you'll need to reference the correct user_id.
+
+## Custom Environment Setup
+
+The **Dogfooding** app is primarily for Stream employees to test the Video SDK internally. You can use one of our environments (prefereably `demo` which is most stable one) for testing, however, it is recommended to configure a custom environment if you have a **Stream app** created in the **Stream Dashboard**.
+
+With **Dogfooding** connected yo your own **Stream app** you will have access to full configuration as well as **Call Stats**, **Events Logs** etc.
+
+Additionaly **APN Push Provider** used for ringing on **iOS devices** is tied to the iOS app **bundle identifier**. Since you will have to change the identifier to run the Dogfooding app using your own Apple Developer account, ringing won't work on iOS.
```

Comment:
```suggestion
Additionaly **APN Push Provider** used for ringing on **iOS devices** is tied to the iOS app **bundle identifier**. Since you will have to change the identifier to run the Dogfooding app using your own Apple Developer account, ringing won't work on iOS with the default environments.
```